src/igvfd/audit/measurement_set.py

```diff
@@ -82,7 +87,11 @@ def audit_seqspec(value, system):
 
 @audit_checker('MeasurementSet', frame='object')
 def audit_unspecified_protocol(value, system):
-    '''MeasurementSet objects should specify the associated link to the protocol for conducting the assay on Protocols.io'''
+    '''
+        audit_detail: Measurement sets should specify the associated link to the protocol for conducting the assay on protocols.io.
```

Comment:
Measurement sets are expected to specify the experimental protocol utilized for conducting the assay on protocols.io.